### ISSUE TITLE
Draft Affordable Care Act Information Page

### ISSUE THREAD

**Issue by LauraTrujillo** [cityofaustin/techstack#1126]
original link: http://austintexas.gov/page/affordable-care-act-grantees-and-service-providers and combine with: http://austintexas.gov/page/affordable-care-act

copy this template (disregard OPO content): https://docs.google.com/document/d/1DAcBVOsWZ0F0nFEIjOFX_-9RxG1jCzMUf4jTzg1O6rI/edit

Save in:
Austin.gov > Content > Content Transitions > Public Health > Information Pages

**Comment by LauraTrujillo** [comment]
i want to talk to the person responsible for this program- not sure what the goals of these pages are...

**Comment by desigonz** [comment]
@LauraTrujillo I took a look at these pages. Some thoughts: 
- On both pages, I checked analytics for outbound links, and they are pretty abysmal (especially on AustinHIV)
- It looks like the [ACA page on the AustinHIV branded site](http://austintexas.gov/page/affordable-care-act-grantees-and-service-providers) targets users who are service providers
  - Let's cut this. 28 pageviews in all of 2018.
- For the [general APH affordable care act page](http://austintexas.gov/page/affordable-care-act), I *think* the target user is any Austin resident interested in signing up for the Affordable Care Act.
  - About 1.4K pageviews in 2018, which is pretty low. I'm not entirely convinced that [this page](http://austintexas.gov/page/affordable-care-act) even needs to exist on Austin.gov. It's not a City of Austin service. It looks like the page hasn't been updated since 2014. 

I recommend not transitioning either. If we get pushback, we can add to the backlog for next round of APH content.

While it's not perfect, I like what [Mass.gov](https://www.mass.gov/information-for-providers-about-the-affordable-care-act-aca) does: provides Massachusetts-specify information on the ACA, but otherwise links. (Although it's linking out to Medicaid.gov, not Healthcare.gov...I'm going to ignore that for the sake of this example.) We *could* treat this content like we're thinking about homelessness assistance content that's partner-provided: use an information page, only highlight need to know information, and link out as quickly as possible, allowing the other side to be the source of truth.
